### PR TITLE
Autogtp skip error message

### LINKED ISSUE #223: GTP error because of OpenCL error messages
I had the following error with Fedora 27. The reason is that Intel Gen OCL Driver does not work correctly on Fedora and prints "cl_get_gt_device(): error, unknown device: 0" to stdout. Autogtp treats the opencl error message as gtp message...

```
autogtp v4
Best network hash: 223737476718d58a4a5b0f317a1eeeb4b38f0c06af5ab65cb9d76d68d9abadb6
Required client version: 4 (OK)
Already downloaded network.
Engine has started.
GTP: cl_get_gt_device(): error, unknown device: 0

*ERROR*: Error in GTP response.
```

### PR DESCRIPTION
**Pull request by ljn917**
Fix #223

### CODE REVIEW

**Comment by ljn917** [comment]
Fix #226

File: autogtp/Game.cpp [hunk 1]
```diff
@@ -65,17 +65,23 @@ void Game::error(int errnum) {
 }
 
 bool Game::eatNewLine() {
-    char readBuffer[256];
+    char readBuffer[256] = {0};
```
Comment (gcp) [inline autogtp/Game.cpp]: Why do you need this if you have readCount?

File: autogtp/Game.cpp [hunk 2]
```diff
@@ -89,6 +95,7 @@ bool Game::sendGtpCommand(QString cmd) {
     char readBuffer[256];
     int readCount = readLine(readBuffer, 256);
     if (readCount <= 0 || readBuffer[0] != '=') {
+        QTextStream(stdout) << "sendGtpCommand: " << cmd << endl;
```
Comment (gcp) [inline autogtp/Game.cpp]: Do we need this? Seems more useful for debugging.

File: autogtp/Game.cpp [hunk 3]
```diff
@@ -65,17 +65,23 @@ void Game::error(int errnum) {
...
         error(Game::PROCESS_DIED);
         return false;
     }
-    auto readCount = readLine(readBuffer, 256);
-    if(readCount < 0) {
-        error(Game::WRONG_GTP);
-        return false;
-    }
+    do {
+        waitForReadyRead(100);
```
Comment (gcp) [inline autogtp/Game.cpp]: Magic delay with a random value? Why is this needed? Why this value? I won't merge this...

**Comment by ljn917** [comment]
I am okay if you do not want to merge those because the real issue is in intel beignet driver. It is just an ugly workaround.

**Comment by ljn917** [comment]
I will just close this because it won't be merged.
